Added contents of CONTRIBUTING.md (#3274)
opengisch/QField: CONTRIBUTING.md

# Contributing to QField

> QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Scroll to bottom if you are a developer.

Reviewer: ```suggestion
QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.
```

> QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.

Reviewer 2: ```suggestion
QField is an open-source project and we appreciate contributions very much. If you are a developer or not, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.
```

## Bug reporting and bug verification

You can help us by submitting bug reports or verification of bugs in the [QField issue tracker](https://github.com/opengisch/QField/issues).

## Documentation

Join the documentation effort about QField. Add, improve or translate the QField documentation in your language.
Check [QField-docs repository](https://github.com/opengisch/QField-docs#qfield-documentation) for more information.

## Translations

> Please help translate QField to your language. Community efforst already made QField available in multiple languages. However, translations are an ongoing effort with the fast pace development of QField, so regularly check your language. Learn more [how to become a translator on Transifex](https://docs.transifex.com/getting-started-1/translators#joining-a-translation-team).

Reviewer 2: ```suggestion
Please help translate QField to your language. Community efforts already made QField available in multiple languages. However, translations are an ongoing effort with the fast pace development of QField, so regularly check your language. Learn more [how to become a translator on Transifex](https://docs.transifex.com/getting-started-1/translators#joining-a-translation-team).
```
